setup/git/010_TestSetup.rb

```diff
@@ -1,9 +1,4 @@
 test_name "Install packages and repositories on target machines..." do
-  extend Beaker::DSL::InstallUtils
-
-  SourcePath  = Beaker::DSL::InstallUtils::SourcePath
-  GitHubSig   = Beaker::DSL::InstallUtils::GitHubSig
```

Comment:
Do we not still need these references defined here?

Reply:
Nope. They're pulled into the DSL namespace, which is set up before pre-suite execution. In fact, these references aren't even complete, since they're relying on a previous chain of inclusion that no longer exists. Both of those constants are actually defined in `FOSSUtils`, which is included in the DSL [here](https://github.com/puppetlabs/beaker-puppet/blob/master/lib/beaker-puppet.rb#L27) via `DSL.register`.

Reply:
That's super lovely